fedimint-server/src/consensus/server.rs

```diff
@@ -668,6 +673,18 @@ impl ConsensusServer {
 
                 Ok(())
             }
+            ConsensusItem::Heartbeat(send_time) => {
+                let now = now();
+                let latency = now.duration_since(send_time).unwrap_or_default();
+
+                debug!(target: LOG_CONSENSUS, "Received heratbeat from {} with latency {} ms sent at {:?}", peer_id, latency.as_millis(), send_time);
+                self.latest_heartbeat_by_peer
+                    .write()
+                    .await
+                    .insert(peer_id, (session_index, now, latency));
+
+                Ok(())
```

Comment:
Should be handled in `process_consensus_item` instead so we can drop the item and not save it to disk.